spec/index.bs

```diff
@@ -3508,7 +3508,9 @@ dictionary GPUSamplerDescriptor : GPUObjectDescriptorBase {
     {{GPUCompareFunction}}.
 - {{GPUSamplerDescriptor/maxAnisotropy}} specifies the maximum anisotropy value clamp used by the sampler.
 
-    Note: most implementations support {{GPUSamplerDescriptor/maxAnisotropy}} values in range between 1 and 16, inclusive.
+    Note: Most implementations support {{GPUSamplerDescriptor/maxAnisotropy}} values in range between 1 and 16, inclusive.
+          The used value of {{GPUSamplerDescriptor/maxAnisotropy}} will be clamped to the maximum value that the platform
+          supports.
```

Comment:
```suggestion
    The used value of {{GPUSamplerDescriptor/maxAnisotropy}} will be clamped to the maximum value that the platform
    supports.
```
"Note:" is text starting a paragraph, so the whole paragraph should be formatted together.